.doc_gen/metadata/cloudwatch_metadata.yaml

```diff
@@ -829,6 +937,15 @@ cloudwatch_GetStartedMetricsDashboardsAlarms:
     - Clean up dashboards, alarms, and anomaly detectors.
   category: Scenarios
   languages:
+    Java:
```

Comment:
The cloudwatch_PutMetricData still has the old snippet. I think it's fine if you want to include both, but the new snippet should be there too, because it shows adding the data to the same metric used in the alarm.